include/rdma/fi_eq.h

```diff
@@ -75,10 +77,18 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+#ifndef _WIN32
 struct fi_mutex_cond {
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
 };
+#else /* _WIN32 */
+struct fi_critsec_cond {
+	CRITICAL_SECTION	*mutex;
```

Comment:
I think we want a different variable name here.  In true windows fashion, maybe change this to:
```c
LPCRITICAL_SECTION    critsec;
```
(I can't bring myself to use CriticalSection as the name. :)